templates/tonic-worker-deployment.yaml

```diff
@@ -60,7 +60,7 @@ spec:
             readOnlyRootFilesystem: true
             {{- end }}
           volumeMounts:
-            {{- if .Values.useUnprivilegedContainers }}
+            {{- if .Values.readOnlyRootFilesystem }}
```

Comment:
Did a sweep through the other files and seems like this is the only place it happened.